[NRF52] Radio Registers broken

It looks like changes in the ```Register Interface``` broke the radio configuration for NRF52 i.e, ```matches``` is removed as I can tell.

```
error[E0599]: no method named `matches` found for type `kernel::common::regs::ReadWrite<u32, radio::Event::Register>` in the current scope
   --> /home/travis/build/helena-project/tock/chips/nrf52/src/radio.rs:678:27
    |
678 |         if regs.event_end.matches(Event::READY::SET) {
    |                           ^^^^^^^
```

--- a/chips/nrf52/src/radio.rs
+++ b/chips/nrf52/src/radio.rs
@@ -661,24 +661,24 @@ impl Radio {
         let regs = unsafe { &*self.regs };
         self.disable_all_interrupts();
 
-        if regs.event_ready.matches(Event::READY::SET) {
+        if regs.event_ready.is_set(Event::READY) {
             regs.event_ready.write(Event::READY::CLEAR);
             regs.event_end.write(Event::READY::CLEAR);
             regs.task_start.write(Task::ENABLE::SET);
         }
 
-        if regs.event_address.matches(Event::READY::SET) {
+        if regs.event_address.is_set(Event::READY) {
             regs.event_address.write(Event::READY::CLEAR);
         }
-        if regs.event_payload.matches(Event::READY::SET) {
+        if regs.event_payload.is_set(Event::READY) {
             regs.event_payload.write(Event::READY::CLEAR);
         }
 
         // tx or rx finished!
-        if regs.event_end.matches(Event::READY::SET) {
+        if regs.event_end.is_set(Event::READY) {
             regs.event_end.write(Event::READY::CLEAR);
 
-            let result = if regs.crcstatus.matches(Event::READY::SET) {
+            let result = if regs.crcstatus.is_set(Event::READY) {
                 ReturnCode::SUCCESS
             } else {
                 ReturnCode::FAIL

--- a/chips/nrf52/src/uart.rs
+++ b/chips/nrf52/src/uart.rs
@@ -308,7 +308,7 @@ impl Uarte {
     /// Check if the UART tranmission is done
     pub fn tx_ready(&self) -> bool {
         let regs = unsafe { &*self.regs };
-        regs.event_endtx.matches(Event::READY::SET)
+        regs.event_endtx.is_set(Event::READY)
     }
 
     fn set_dma_pointer_to_buffer(&self) {

